django_wikidata_api/models.py

```diff
@@ -35,6 +35,7 @@ class Meta(object):
 
     main = WikidataEntityField(triples=[], required=True)
     label = WikidataLabelField(required=True)
+    description = WikidataDescriptionField()
```

Comment:
Here, I'm adding the description field to the Base model, so now by default all queries will pull in the description along with the label and qid